Client/Frontend/Browser/Onboarding/Welcome/ReleaseNotesView.swift

```diff
@@ -0,0 +1,94 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveShared
+
+struct ReleaseNotesView: View {
+    @Environment(\.colorScheme) var colorScheme: ColorScheme
+
+    var dismiss: (() -> Void)?
+
+    var body: some View {
+        VStack {
+            Image(uiImage: #imageLiteral(resourceName: "brave-vpn-ad-top-shimmer"))
+            .resizable()
+            .aspectRatio(contentMode: .fit)
+            HStack {
+                Spacer()
+                Button {
+                    dismiss?()
+                } label: {
+                    Image(uiImage: #imageLiteral(resourceName: "privacy-everywhere-exit-icon"))
+                }
+                .padding(.trailing, 10)
+                .padding(.top, 10)
+            }
```

Comment:
Change this to 
```swift
Button(...)
  .padding(...)
  .padding(...)
  .frame(maxWidth: .infinity, alignment: .trailing)
```

Reply:
Same as https://github.com/brave/brave-ios/pull/4508#discussion_r750624936